$NetBSD: patch-Makefile.am,v 1.2 2024/06/18 18:23:15 bsiegert Exp $

Add --tag=CXX libtool flag on NetBSD and Dragonfly.
Upstream pull req: https://github.com/meganz/MEGAcmd/pull/949

--- Makefile.am.orig	2024-05-20 12:29:43.000000000 +0000
+++ Makefile.am
@@ -39,6 +39,14 @@ if LINUX
 AM_LIBTOOLFLAGS="--tag=CXX"
 endif
 
+if DRAGONFLY
+AM_LIBTOOLFLAGS="--tag=CXX"
+endif
+
+if NETBSD
+AM_LIBTOOLFLAGS="--tag=CXX"
+endif
+
 VERSION_INFO=$(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
 
 CLEANFILES=
